app/services/SubmissionService.scala

```diff
@@ -50,10 +50,10 @@ class SubmissionService @Inject()(
         auditService.auditAllPagesUserInput(choice, cacheMap)
 
         val movementAuditType =
-          if (choice.value == Choice.AllowedChoiceValues.Arrival) AuditTypes.AuditArrival else AuditTypes.AuditDeparture
+          if (choice == Arrival) AuditTypes.AuditArrival else AuditTypes.AuditDeparture
```

Comment:
Is AuditTypes is defined for every choice ?  If yes maybe would be nice to link it as Choice filed.

Reply:
Nope, it's defined, but the logic for it is just related with Arrival and Departure